lang/en.json

```diff
@@ -211,6 +211,8 @@
         "FileIntegrityWarningReuploadBis": "Try to reupload all the Matomo files in BINARY mode.",
         "First": "First",
         "Flatten": "Flatten",
+        "ForcedSSL": "Forced SSL Connection",
+        "ForceSSLRecommended": "We recommend using Matomo over secure SSL connections only. To disallow unsecure access over http add `force_ssl = 1` to `General` section in Matomo config.",
```

Comment:
Think `force_ssl = 1` and `General` don't need to be translated correct? And are the backticks supposed to be there or should there be `<code>` elements (inserted via placeholder)?

Reply:
Would change "disallow unsecure" to "prevent insecure", otherwise sounds good to me.

Reply:
changed.